src/api.js

```diff
@@ -163,16 +165,19 @@ api.declare({
       msg.organization = sanitizeGitHubField(body.repository.owner.login),
       msg.action = body.action;
       msg.details = getPullRequestDetails(body);
+      msg.installationID = body.installation.id;
```

Comment:
These messages end up in a publicly readable stream on pulse. Are they something we should be keeping more secret? I'm thinking that we should probably do the following (although probably not in this PR):

1. Remove the installationId from this message and just publish it as it was before.
2. Don't have the worker in a separate process that is listening on pulse. Let's just do that work in the web endpoint. We can still publish messages in case anybody wants them, but it is not needed for the worker afaict.